runtime/v1/linux/bundle.go

```diff
@@ -179,6 +179,9 @@ func (b *bundle) shimConfig(namespace string, c *Config, runcOptions *runctypes.
 
 // atomicDelete renames the path to a hidden file before removal
 func atomicDelete(path string) error {
+	if _, err := os.Stat(path); err != nil && os.IsNotExist(err) {
+		return nil
```

Comment:
I think we can just check for IsNotExist on rename instead of making an extra stat call.

Reply:
Also think this way, the current way can avoid doing `RemoveAll`

Reply:
can we handle this error on the `os.Rename`?